packages/postcss-normalize-repeat-style/src/__tests__/index.js

```diff
@@ -38,10 +35,17 @@ function suite(fixture, expected) {
     ]);
 }
 
-Object.keys(data).forEach((conversion) => {
-  const fixture = data[conversion];
-
-  test(fixture, suite(fixture, conversion));
+[
```

Comment:
Same as for the other test, this test could pass even if the mapping in the main code was wrong.